### PR TITLE
Use new SDK version (v1.18.3)

### PR DESCRIPTION
**Pull request by chkp-eladya**

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -56,7 +56,7 @@ github.com/bgentry/go-netrc/netrc
 github.com/bgentry/speakeasy
 # github.com/davecgh/go-spew v1.1.1
 github.com/davecgh/go-spew/spew
-# github.com/dome9/dome9-sdk-go v1.18.1 => ../dome9-sdk-go
+# github.com/dome9/dome9-sdk-go v1.18.3
```
Comment (Benjamin-sh-ckpnt) [inline vendor/modules.txt:59]: I think this was here for local debugging, (commented out)?
maybe just replace to 1.18.3 and add comment that this is how to use locally?

File: go.sum [hunk 2]
```diff
@@ -37,6 +37,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dome9/dome9-sdk-go v1.18.1 h1:M1+QMbUXhlFS1cHEyJEL8COAbMNSLqm3oAgXBVE5OIY=
 github.com/dome9/dome9-sdk-go v1.18.1/go.mod h1:CF7nXCQk74ApsoG2i1+ziC/pnnnsyMeG6Mpho8cRAGM=
+github.com/dome9/dome9-sdk-go v1.18.3 h1:KqGg1OQzAxLmwTCkx64GZU8J/mrCeZKYrWfOyyVFn1A=
+github.com/dome9/dome9-sdk-go v1.18.3/go.mod h1:CF7nXCQk74ApsoG2i1+ziC/pnnnsyMeG6Mpho8cRAGM=
```
Comment (Benjamin-sh-ckpnt) [inline go.sum]: do we still need the old v1.18.1 lines?

**Review by Benjamin-sh-ckpnt — COMMENTED** [review]
what's the deal with the code under vendor/github....?

isn't that the exact same code you changed in the other repo?

**Comment by chkp-eladya** [comment]
> what's the deal with the code under vendor/github....?
> 
> isn't that the exact same code you changed in the other repo?

Yes it is.
When you run the command "go mod vendor" it pulls the dependencies and stores them locally under "vendor" dir.
It seems that they were working this way in all the previous PRs also

File: vendor/modules.txt (hunk 1 above)
Comment (chkp-eladya) [reply vendor/modules.txt:59]: This file is being modified automatically when I use the "go mod vendor" command.
I don't have the ability to change it manually:
image

I think that this is the right way it should be, according to previous commits examples:
https://github.com/dome9/terraform-provider-dome9/commit/3a0d3782f65a4d1a8d63b06da8c4388c3ce98af3
https://github.com/dome9/terraform-provider-dome9/commit/1b175268de60665d47c80778f0dc8a5b7405f58d

File: go.sum (hunk 2 above)
Comment (chkp-eladya) [reply go.sum]: fixed

**Review by Benjamin-sh-ckpnt — APPROVED** [review]
LGTM!

**Review by alexgill-cp — APPROVED** [review]

**Pull request merged**